gcp/bedrock-demos/cloudrun/mozorg-demo.env.yaml

```diff
@@ -11,7 +11,8 @@ ALLOWED_HOSTS: .allizom.org,.moz.works,.run.app
 CONTENT_CARDS_URL: https://www-dev.allizom.org/media/
 CSP_DEFAULT_SRC: "*.allizom.org"
 CSP_EXTRA_FRAME_SRC: "*.mozaws.net,o1069899.sentry.io"
-CSP_REPORT_ENABLE: "True"
+CSP_REPORT_URI: https://o1069899.ingest.us.sentry.io/api/6260338/security/?sentry_key=97ec0cd426714b728e92f3b3aa62f00b
+CSP_REPORT_PERCENTAGE: 75
```

Comment:
Once there are both enforced and report-only present at the same time, would they share the config (i.e. endpoint and reporting %%), or one or both are gonna need to be set differently? Mostly I'm thinking if it's possible to differentiate the real violations in Sentry from just the "soft" report-only events from the payloads and separate them accordingly, or if there's a need to gather the two in their own sets to differentiate real CSP issues from just future/projected changes' impact.

(Right now there's only the enforced configuration present, do you plan to also include some report-only config with this changeset to factor in the possibility of two sets of reporting prefs?)

BTW what would be the plan for enforced+report-only setup in general? Run report-only just for demo/staging etc. to gather some stats there, or deploy it also to prod with reasonably small %% sample to get real traffic data? (Can the percentage in that case be smaller than 1% or that's the lowest value?)

Reply:
more discussion is needed on these exact questions. I am thinking that keeping everything separated would be cleanest. I'm not sure how Sentry can segregate the different types. I'm also not sure if Sentry can offer 2 separate reporting URLs. But until we have a report-only CSP we can chug along with the enforced one and figure that out later.

Reply:
I tried to search their docs if/how they segregate ingests based on payload [`disposition`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only#disposition) but to no avail. Maybe it's obvious once the data's there, but right now I don't see any good best practice to refer to.